pkgs/tools/networking/ht-rust/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, rustPlatform, fetchFromGitHub, Security }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ht-rust";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "ducaale";
+    repo = "ht";
+    rev = "v${version}";
+    sha256 = "cr/iavCRdFYwVR6Iemm1hLKqd0OFG1iDmxpQ9fiwOmU=";
+  };
+
+  cargoSha256 = "uB23/9AjPwCwf9ljE8ai7zJQZqE0SoBPzRqqBOXa9QA=";
+
+  buildInputs = [ ] ++ lib.optional stdenv.isDarwin Security;
+
+  # Symlink to avoid conflict with pre-existing ht package
```

Comment:
Doesn't this still conflict? I guess you wanted to rename `ht` to `ht-rust`?

Reply:
Yes and no. Usually our nix profiles will ignore conflicts and prefer one package. Users can give packages however priority using `lib.hiPrio`. Using something like `lib.hiPrio ht` and `ht-rust` would allow to use both executables at the same time via ht-rust and ht